go/vt/topotools/keyspace.go

```diff
@@ -121,7 +137,7 @@ func UpdateShardRecords(
 		// For 'to' shards, refresh to make them serve. The 'from' shards will
 		// be refreshed after traffic has migrated.
 		if !isFrom {
-			if _, err := RefreshTabletsByShard(ctx, ts, tmc, si, cells, logger); err != nil {
+			if _, _, err := RefreshTabletsByShard(ctx, ts, tmc, si, cells, logger); err != nil {
```

Comment:
This is called from a number of places via `Wrangler.updateShardRecords()`. I don't think that we can change the partial handling here w/o other changes.